setup.py

```diff
@@ -64,17 +66,23 @@
 
 INSTALL_REQUIRES = ['pyparsing >= 2.0.1', 'pyperclip', 'six']
 
-# Windows also requires pyreadline to ensure tab completion works
-if sys.platform.startswith('win'):
-    INSTALL_REQUIRES += ['pyreadline']
-
-# Python 3.4 and earlier require contextlib2 for temporarily redirecting stderr and stdout
-if sys.version_info < (3, 5):
-    INSTALL_REQUIRES += ['contextlib2']
+EXTRAS_REQUIRE = {
```

Comment:
I wasn't familiar with this syntax within setuptools for conditional dependencies.